lib/otel/rules.js

```diff
@@ -136,10 +159,16 @@ class RulesEngine {
   test(otelSpan) {
```

Comment:
I really don't know what to do to silence this lint error. I tried returning the result instead of assigning and then returning, but that didn't help.

Reply:
it's because there's logic in multiple case statements. it's fine for now. i'd just ignore this rule at the function scope for now

Reply:
Sold!